.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,7 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/go
 {
 	"name": "Go",
-	"image": "mcr.microsoft.com/devcontainers/go:0-1.21-bullseye",
+	"image": "mcr.microsoft.com/devcontainers/go:1.22-bookworm",
```

Comment:
Could you use a Go 1.21 image? We set the base version as the lowest supported Go version, and we bump it whenever there is an update in the Go EOL (https://endoflife.date/go)

Reply:
@mdelapenya Thank you for reviewing my changes.
I've changed the Go version, so could you please take another look?